rosidl_typesupport_introspection_c/rosidl_typesupport_introspection_c/cli.py

```diff
@@ -25,7 +25,10 @@
 
 
 class GenerateIntrospectionCTypesupport(GenerateCommandExtension):
+    """
+    Generator class for generating the C implementation of the type support.
```

Comment:
CI is failing because pep257 thinks this can all be on one line.

Reply:
let's see if this is enough c41fbcdb766e1b99288502509b5d4ef86273b98e